### ISSUE TITLE
The language switch menu is getting to large for mobile

### ISSUE THREAD

**Issue by maccyber** [rubynor/bigfive-web#61]
The language switch menu is getting to large for mobile-views.

Possible fixed on [sm-breakpoint](https://vuetifyjs.com/en/features/breakpoints/#display-breakpoints):
- Fullscreen menu
- Decrease padding around menu elements
- 3 (4?) cols 

The component is located at https://github.com/rubynor/bigfive-web/blob/master/components/LanguageSwitcher.vue

For vuetify-documentation on the menu component:
https://vuetifyjs.com/en/components/menus/#menus
https://vuetifyjs.com/en/api/v-menu/

![menu](https://user-images.githubusercontent.com/701351/95388724-0a6e2d80-08f3-11eb-91d5-db09f510672b.png)

**Comment by mcastre** [comment]
Hi @maccyber i can knock this out for you!

**Comment by maccyber** [comment]
@mcastre: Are you still on this?

**Comment by mcastre** [comment]
Yes I’m still working on it. Got a bit more complicated to make the fix work on mobile and desktop
